src/site/content/en/blog/content-indexing-api/index.md

```diff
@@ -5,28 +5,19 @@ authors:
   - jeffposnick
 description: Your PWA might cache articles and media files, but how will your users know that your pages work while offline? The Content Indexing API is one answer to this question currently in an origin trial. Once the index is populated with content from your PWA, as well as any other installed PWAs, it will show up in dedicated areas of supported browsers.
 date: 2019-12-12
-updated: 2020-05-07
+updated: 2020-05-26
```

Comment:
Just double-checking that you intended May 26. I assume that's timed for M84.

Reply:
The only other real choice is the 27th. I see no reason to wait. I'd be fine with pushing it today if it were ready.

Reply:
M84 is beta on Thursday. I intended this to go out this week. I'm linking to it from the beta release post.